app/app.js

```diff
@@ -126,7 +126,30 @@ function (app, $, _, Backbone, Bootstrap, Helpers, Utils, FauxtonAPI, Couchdb) {
           type: 'REMOVE_NAVBAR_LINK',
           link: link
       });
+    },
+
+    isRunningOnBackdoorPort: function () {
```

Comment:
I think the name is too broad. If...when.../_config will be available on cluster interface it will be easy to break all the logic behind that name. 

I would suggest to use some specific like "isConfigAvailable" to control features availability per resource basis or use some cluster-only resource like `_membership` or `_cluster_setup` e.g. those that only makes a sense for cluster interface. 

Alternatively, you may figure if you're on cluster or not from `GET /` request - cluster has no uuid in response, but that's may be also temporary solution.